src/overlays/actors/ovl_En_Famos/z_en_famos.c

```diff
@@ -366,7 +366,7 @@ void EnFamos_StillIdle(EnFamos* this, PlayState* play) {
 void EnFamos_SetupPathingIdle(EnFamos* this) {
     if (this->isCalm) {
         this->currentPathNode++;
-        if (this->currentPathNode == this->pathNodeCount) {
+        if (this->currentPathNode == this->pathCount) {
             this->currentPathNode = 0;
```

Comment:
```suggestion
        if (this->pathIndex == this->pathCount) {
            this->pathIndex = 0;
```

Reply:
Not really the `pathIndex`, it is the `waypointIndex` (current point on the path)